install.sh

```diff
@@ -200,6 +200,34 @@ file_not_grpowned() {
   [[ " $(id -G "$USER") " != *" $(get_group "$1") "*  ]]
 }
 
+old_ruby() {
+  if builtin command -v ruby &>/dev/null; then
+    local ruby_version
+    ruby_version=$(ruby -e 'puts RUBY_VERSION' | grep -o '^[0-9]\+\.[0-9]\+')
```

Comment:
Use logic like this: https://github.com/Homebrew/brew/blob/53fe2c94dbfecf78192fd05b7c3adc245e67f8d6/Library/Homebrew/utils/ruby.sh#L56

Reply:
What advantage does that approach have?

Reply:
Consistency is the main advantage. Otherwise there are cases where the version in this PR will fail and the version in Homebrew/brew will not (or vice-versa).

Reply:
Suggestion implemented. Resolving the conversation to cleanup the page.